lib/src/main/java/grpcbridge/route/Route.java

```diff
@@ -23,13 +25,16 @@
 public final class Route {
     private final MethodDescriptor descriptor;
     private final ServerMethodDefinition<Message, Message> impl;
+    private final List<Parser> parsers;
```

Comment:
Not sure why we need it per route, because the accepted message type is controlled by the client.

Reply:
The body parser needs to be aware of the parser that it is using, these parsers are registered on the bridge builder but needs to be passed down. I'm not sure how to get around that, i don't think it's really possible.